src/views/layouts/application.slang

```diff
@@ -35,5 +36,6 @@ html
         == render_template "layouts/_footer.slang"
 
     script src="/javascripts/main.js"
+    script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"
```

Comment:
As far as i understand we need to create a custom package to support Crystal.

https://highlightjs.org/download/